uber/site_sections/registration.py

```diff
@@ -32,7 +32,7 @@ def checking_at_the_door(self, *args, **kwargs):
 
 @all_renderable(c.PEOPLE, c.REG_AT_CON)
 class Root:
-    def index(self, session, message='', page='0', search_text='', uploaded_id='', order='last_first', invalid=''):
+    def index(self, session, message='', page='1', search_text='', uploaded_id='', order='last_first', invalid=''):
```

Comment:
would you be opposed if i added something different at the top of this like:

```
if c.DEV_BOX and (not page or page < 1):
    page = 1
```

mostly during testing it's just annoying to have to keep clicking through to 'page 1', and we don't care about perf there.